<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);</string>
      </property>
      <widget class="QLabel" name="lblFavoriteCount">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>250</y>
         <width>291</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lblUsername">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>160</y>
         <width>291</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Open Sans ExtraBold</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btnBack">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>480</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>thoát</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtNewPassword">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>380</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtNewEmail">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>320</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Email</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnChangeEmail">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>430</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>đổi email</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../image/gmail.png</normaloff>../image/gmail.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="btnChangePassword">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>430</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>đổi mật khẩu</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../image/padlock.png</normaloff>../image/padlock.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="btnDeleteAccount">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>480</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>xoá tài khoản</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblAvatar">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>120</width>
         <height>120</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>120</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btnChangeAvatar">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>60</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>đổi avatar</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblPassword">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>210</y>
         <width>291</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Open Sans ExtraBold</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);</string>
      </property>
      <widget class="QListWidget" name="listFavorites">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>271</width>
         <height>431</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
